src/Service/MigrationService.php

```diff
@@ -111,8 +111,9 @@ public function getNewMigrations(): array
     {
         $applied = [];
 
+        /** @psalm-var string $class */
```

Comment:
```suggestion
        /** @psalm-var string $class */
```

Add annotation to `Migrator::getHistory()` instead.